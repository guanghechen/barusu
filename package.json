{
  "name": "@barusu/root",
  "version": "0.0.0",
  "author": {
    "name": "guanghechen",
    "url": "https://github.com/guanghechen/"
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/guanghechen/barusu.git"
  },
  "homepage": "https://github.com/guanghechen/barusu/#readme",
  "license": "MIT",
  "private": true,
  "engines": {
    "node": ">= 12.16.0"
  },
  "scripts": {
    ":publish": "yarn format && yarn test && lerna publish",
    "build": "lerna run --concurrency 1 build",
    "clean": "lerna clean --yes && yarn install",
    "format": "run-s format:find-inconsistent format:sort-imports format:lint:fix format:prettier",
    "format:prettier": "prettier --write .",
    "format:find-inconsistent": "node ./node_modules/@barusu/tool-find-inconsistent/lib/cjs/cli.js .",
    "format:lint:fix": "eslint . --fix",
    "format:sort-imports": "node ./node_modules/@barusu/tool-sort-imports/lib/cjs/cli.js .",
    "new:ts-package": "node ./node_modules/@barusu/template-ts-package/cli.js",
    "prepublishOnly": "pinst --disable",
    "postinstall": "run-p postinstall:**",
    "postinstall:main": "yarn update:lock",
    "postinstall:dev": "is-ci || husky install",
    "rebuild": "yarn clean && yarn build",
    "test": "lerna exec --concurrency 1 'npm run test --if-present'",
    "update:changelog": "gitmoji-changelog update",
    "update:lock": "yarn-deduplicate"
  },
  "workspaces": [
    "packages/*",
    "scaffolds/*",
    "templates/*",
    "tools/*",
    "utils/*"
  ],
  "devDependencies": {
    "@guanghechen/rollup-config": "^1.0.2",
    "@types/fs-extra": "^9.0.6",
    "@types/chai": "^4.2.11",
    "@types/jest": "^26.0.19",
    "@types/rimraf": "^3.0.0",
    "@types/supertest": "^2.0.10",
    "chai": "^4.2.0",
    "chai-exclude": "^2.0.2",
    "cross-env": "^7.0.3",
    "fs-extra": "^9.0.1",
    "gitmoji-changelog": "^2.2.0",
    "husky": "^5.0.9",
    "is-ci": "^3.0.0",
    "lerna": "^4.0.0",
    "lint-staged": "^10.5.4",
    "jest": "^26.6.3",
    "mocha": "^8.1.1",
    "npm-run-all": "^4.1.5",
    "pinst": "^2.1.6",
    "rimraf": "^3.0.2",
    "rollup": "2.39.1",
    "supertest": "^4.0.2",
    "ts-jest": "^26.4.4",
    "ts-node": "~9.1.1",
    "tsconfig-paths": "^3.9.0",
    "typescript": "~4.1.5",
    "yarn-deduplicate": "^3.1.0"
  },
  "@barusu/tool-sort-imports": {
    "logLevel": "verbose",
    "pattern": [
      "{packages,scaffolds,templates,tools,utils}/*/{src,test,__test__}/*.{ts,tsx}",
      "{packages,scaffolds,templates,tools,utils}/*/src/**/*.{ts,tsx}"
    ],
    "moduleRanks": [
      {
        "regex": "^(react|vue|angular)(?:[/\\-][\\w\\-./]*)?$",
        "rank": 1.1
      },
      {
        "regex": "^mocha|chai(?:[/][\\w\\-./]*)?$",
        "rank": 1.2
      },
      {
        "regex": "^[a-zA-Z\\d][\\w\\-.]*",
        "rank": 1.3
      },
      {
        "regex": "^@[a-zA-Z\\d][\\w\\-.]*\\/[a-zA-Z\\d][\\w\\-.]*",
        "rank": 1.4
      },
      {
        "regex": "^@\\/",
        "rank": 2.1
      },
      {
        "regex": "^(?:\\/|[a-zA-Z]:)",
        "rank": 3.1
      },
      {
        "regex": "^[.]{2}[\\/\\\\][^\\n]*",
        "rank": 3.2
      },
      {
        "regex": "^[.][\\/\\\\][^\\n]*",
        "rank": 3.3
      }
    ],
    "indent": "  ",
    "quote": "'",
    "semicolon": false,
    "maxColumn": 80,
    "blankLinesAfterStatement": 1
  },
  "lint-staged": {
    "*.{js,jsx,ts,tsx}": [
      "eslint --fix",
      "prettier --write"
    ]
  }
}
