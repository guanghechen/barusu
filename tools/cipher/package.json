{
  "name": "@barusu/tool-cipher",
  "version": "0.0.19",
  "description": "cipher data",
  "author": {
    "name": "lemon-clown",
    "url": "https://github.com/lemon-clown/"
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/lemon-clown/barusu.git"
  },
  "homepage": "https://github.com/lemon-clown/barusu/tree/master/tools/cipher#readme",
  "main": "lib/cjs/index.js",
  "types": "lib/types/index.d.ts",
  "source": "src/index.ts",
  "license": "MIT",
  "engines": {
    "node": ">= 12.16.0"
  },
  "files": [
    "lib/",
    ".eslintrc.js",
    "CHANGELOG.md",
    "LICENSE",
    "README.md",
    "package.json",
    "!lib/**/*.js.map",
    "!lib/**/*.d.ts.map"
  ],
  "bin": {
    "barusu-cipher": "./lib/cjs/cli.js"
  },
  "scripts": {
    "start": "rollup -w -c rollup.config.js",
    "prebuild": "yarn test && rimraf lib/",
    "build": "rollup -c rollup.config.js",
    "test": "echo 'no test yet.'",
    "prepublishOnly": "cross-env ROLLUP_USE_SOURCE_MAP=false yarn build",
    "lint:fix": "eslint 'src/**/*.{ts,tsx}' --fix"
  },
  "dependencies": {
    "@barusu/chalk-logger": "^0.0.19",
    "@barusu/event-bus": "^0.0.19",
    "@barusu/util-cli": "^0.0.19",
    "@barusu/util-option": "^0.0.19",
    "@types/cli-progress": "^3.7.0",
    "@types/fs-extra": "^8.1.0",
    "cli-progress": "^3.8.2",
    "commander": "^5.0.0",
    "fs-extra": "^8.1.0",
    "globby": "^11.0.0"
  },
  "devDependencies": {
    "@barusu/eslint-config": "^0.0.19",
    "@barusu/rollup-config": "^0.0.19",
    "@types/chai": "^4.2.11",
    "@types/mocha": "^7.0.2",
    "chai": "^4.2.0",
    "cross-env": "^7.0.2",
    "mocha": "^7.1.1",
    "rollup": "2.16.1"
  }
}
