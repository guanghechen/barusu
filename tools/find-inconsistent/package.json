{
  "name": "@barusu/tool-find-inconsistent",
  "version": "0.1.0",
  "description": "A cli tool to find inconsistent package versions in monorepo",
  "author": {
    "name": "guanghechen",
    "url": "https://github.com/guanghechen/"
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/guanghechen/barusu.git",
    "directory": "tools/find-inconsistent"
  },
  "homepage": "https://github.com/guanghechen/barusu/tree/master/tools/find-inconsistent#readme",
  "main": "lib/cjs/index.js",
  "types": "lib/types/index.d.ts",
  "source": "src/index.ts",
  "license": "MIT",
  "engines": {
    "node": ">=14.15.0"
  },
  "files": [
    "lib/",
    "!lib/**/*.js.map",
    "!lib/**/*.d.ts.map",
    "package.json",
    "CHANGELOG.md",
    "LICENSE",
    "README.md"
  ],
  "bin": {
    "barusu-find-inconsistent": "./lib/cjs/cli.js"
  },
  "scripts": {
    "build": "cross-env NODE_ENV=production rollup -c rollup.config.js",
    "prebuild": "rimraf lib/",
    "prepublishOnly": "cross-env ROLLUP_SHOULD_SOURCEMAP=false yarn build"
  },
  "dependencies": {
    "@barusu/chalk-logger": "^0.1.0",
    "@barusu/util-cli": "^0.1.0",
    "@guanghechen/locate-helper": "^1.0.13",
    "@types/fs-extra": "^9.0.8",
    "commander": "^5.1.0",
    "fs-extra": "^9.1.0"
  }
}
