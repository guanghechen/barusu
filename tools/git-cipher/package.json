{
  "name": "@barusu/tool-git-cipher",
  "version": "0.0.28",
  "description": "cipher git repository",
  "author": {
    "name": "lemon-clown",
    "url": "https://github.com/lemon-clown/"
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/lemon-clown/barusu.git"
  },
  "homepage": "https://github.com/lemon-clown/barusu/tree/master/tools/git-cipher#readme",
  "keywords": [
    "cipher tool"
  ],
  "main": "lib/cjs/index.js",
  "types": "lib/types/index.d.ts",
  "source": "src/index.ts",
  "license": "MIT",
  "engines": {
    "node": ">= 12.16.0"
  },
  "files": [
    "lib/",
    "CHANGELOG.md",
    "LICENSE",
    "README.md",
    "package.json",
    "!lib/**/*.js.map",
    "!lib/**/*.d.ts.map"
  ],
  "bin": {
    "barusu-git-cipher": "./lib/cjs/cli.js"
  },
  "scripts": {
    "start": "rollup -w -c rollup.config.js",
    "prebuild": "yarn test && rimraf lib/",
    "build": "rollup -c rollup.config.js",
    "test": "echo 'no test yet.'",
    "prepublishOnly": "cross-env ROLLUP_USE_SOURCE_MAP=false yarn build",
    "lint:fix": "eslint 'src/**/*.{ts,tsx}' --fix"
  },
  "dependencies": {
    "@barusu/chalk-logger": "^0.0.28",
    "@barusu/event-bus": "^0.0.28",
    "@barusu/util-cli": "^0.0.28",
    "@barusu/util-option": "^0.0.28",
    "@types/fs-extra": "^8.1.0",
    "@types/inquirer": "^6.5.0",
    "command-exists": "^1.2.9",
    "execa": "^4.0.3",
    "fs-extra": "^8.1.0",
    "globby": "^11.0.0",
    "inquirer": "^7.3.0",
    "node-plop": "~0.26.0"
  },
  "devDependencies": {
    "@barusu/eslint-config": "^0.0.28",
    "@barusu/rollup-config": "^0.0.28",
    "@barusu/rollup-plugin-copy": "^0.0.28",
    "rollup": "2.16.1"
  }
}
