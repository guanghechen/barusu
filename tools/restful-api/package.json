{
  "name": "@barusu/tool-restful-api",
  "version": "0.0.28",
  "description": "quickly create restful style mock server.",
  "author": {
    "name": "lemon-clown",
    "url": "https://github.com/lemon-clown/"
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/lemon-clown/barusu.git"
  },
  "homepage": "https://github.com/lemon-clown/barusu/tree/master/tools/restful-api#readme",
  "main": "lib/cjs/index.js",
  "types": "lib/types/index.d.ts",
  "source": "src/index.ts",
  "license": "MIT",
  "engines": {
    "node": ">= 12.16.0"
  },
  "keywords": [
    "restful mock server",
    "typescript/json-schema to mock server"
  ],
  "files": [
    "lib/",
    "CHANGELOG.md",
    "LICENSE",
    "README.md",
    "package.json",
    "!lib/**/*.js.map",
    "!lib/**/*.d.ts.map"
  ],
  "bin": {
    "barusu-rapit": "./lib/cjs/cli.js"
  },
  "scripts": {
    "prebuild": "yarn test && rimraf lib/",
    "build": "rollup -c rollup.config.js",
    "test": "TS_NODE_FILES=true jest --config ../../jest.config.json --rootDir .",
    "prepublishOnly": "cross-env ROLLUP_USE_SOURCE_MAP=false yarn build",
    "lint:fix": "eslint src/**/*.ts --fix",
    "generate:schema": "TS_NODE_FILES=true cross-env node -r ts-node/register script/generate.schema.ts",
    "generate:raw-schema": "TS_NODE_FILES=true cross-env node -r ts-node/register script/generate.raw-schema.ts"
  },
  "dependencies": {
    "@barusu/chalk-logger": "^0.0.28",
    "@barusu/configuration-master": "^0.0.28",
    "@barusu/event-bus": "^0.0.28",
    "@barusu/typescript-json-schema": "^0.0.28",
    "@barusu/util-cli": "^0.0.28",
    "@barusu/util-option": "^0.0.28",
    "@koa/cors": "^3.1.0",
    "@koa/router": "^9.3.1",
    "@types/fs-extra": "^8.1.0",
    "@types/inquirer": "^6.5.0",
    "@types/js-yaml": "^3.12.5",
    "@types/json-schema-faker": "^0.5.0",
    "@types/koa": "^2.11.1",
    "@types/koa-json": "^2.0.18",
    "@types/koa-send": "^4.1.2",
    "@types/koa__cors": "^3.0.1",
    "@types/koa__router": "^8.0.2",
    "@types/mockjs": "^1.0.2",
    "execa": "^4.0.3",
    "fs-extra": "^8.1.0",
    "globby": "^11.0.0",
    "inquirer": "^7.3.0",
    "js-yaml": "^3.14.0",
    "json-schema-faker": "^0.5.0-rcv.24",
    "koa": "^2.13.0",
    "koa-json": "^2.0.2",
    "koa-send": "^5.0.1",
    "mockjs": "^1.1.0",
    "typescript": "^3.8.2"
  },
  "devDependencies": {
    "@barusu/eslint-config": "^0.0.28",
    "@barusu/rollup-config": "^0.0.28",
    "@barusu/rollup-plugin-copy": "^0.0.28",
    "rollup": "2.16.1"
  }
}
