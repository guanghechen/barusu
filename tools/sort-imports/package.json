{
  "name": "@barusu/tool-sort-imports",
  "version": "0.0.43",
  "description": "sort import/export statement",
  "author": {
    "name": "guanghechen",
    "url": "https://github.com/guanghechen/"
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/guanghechen/barusu.git"
  },
  "homepage": "https://github.com/guanghechen/barusu/tree/master/tools/sort-imports#readme",
  "main": "lib/cjs/index.js",
  "types": "lib/types/index.d.ts",
  "source": "src/index.ts",
  "license": "MIT",
  "engines": {
    "node": ">= 12.16.0"
  },
  "files": [
    "lib/",
    "CHANGELOG.md",
    "LICENSE",
    "README.md",
    "package.json",
    "!lib/**/*.js.map",
    "!lib/**/*.d.ts.map"
  ],
  "bin": {
    "barusu-sort-imports": "./lib/cjs/cli.js"
  },
  "scripts": {
    "start": "rollup -w -c rollup.config.js",
    "prebuild": "rimraf lib/",
    "build": "cross-env NODE_ENV=production rollup -c rollup.config.js",
    "prepublishOnly": "cross-env ROLLUP_USE_SOURCE_MAP=false yarn build",
    "test": "cross-env TS_NODE_FILES=true jest --config ../../jest.config.js --rootDir .",
    "lint:fix": "eslint '{src,__test__}/**/*.{ts,tsx}' --fix"
  },
  "dependencies": {
    "@barusu/chalk-logger": "^0.0.43",
    "@barusu/event-bus": "^0.0.43",
    "@barusu/util-cli": "^0.0.43",
    "@barusu/util-option": "^0.0.43",
    "@types/fs-extra": "^8.1.0",
    "fs-extra": "^8.1.0",
    "globby": "^11.0.0"
  },
  "devDependencies": {
    "@barusu/eslint-config": "^0.0.43",
    "@barusu/rollup-config": "^0.0.43",
    "@barusu/util-jest": "^0.0.43",
    "rollup": "2.28.1"
  }
}
