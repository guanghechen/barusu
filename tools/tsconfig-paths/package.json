{
  "name": "@barusu/tool-tsconfig-paths",
  "version": "0.0.31",
  "description": "find inconsistent package versions in monorepo",
  "author": {
    "name": "lemon-clown",
    "url": "https://github.com/lemon-clown/"
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/lemon-clown/barusu.git"
  },
  "homepage": "https://github.com/lemon-clown/barusu/tree/master/tools/tsconfig-paths#readme",
  "main": "lib/cjs/index.js",
  "types": "lib/types/index.d.ts",
  "source": "src/index.ts",
  "license": "MIT",
  "engines": {
    "node": ">= 12.16.0"
  },
  "files": [
    "lib/",
    "CHANGELOG.md",
    "LICENSE",
    "README.md",
    "package.json",
    "!lib/**/*.js.map",
    "!lib/**/*.d.ts.map"
  ],
  "bin": {
    "barusu-tsconfig-paths": "./lib/cjs/cli.js"
  },
  "scripts": {
    "start": "rollup -w -c rollup.config.js",
    "prebuild": "yarn test && rimraf lib/",
    "build": "rollup -c rollup.config.js",
    "test": "echo 'no test yet.'",
    "prepublishOnly": "cross-env ROLLUP_USE_SOURCE_MAP=false yarn build",
    "lint:fix": "eslint 'src/**/*.{ts,tsx}' --fix"
  },
  "dependencies": {
    "@barusu/chalk-logger": "^0.0.31",
    "@barusu/tool-sort-imports": "^0.0.31",
    "@barusu/util-cli": "^0.0.31",
    "@barusu/util-option": "^0.0.31",
    "@types/fs-extra": "^8.1.0",
    "fs-extra": "^8.1.0",
    "tsconfig": "^7.0.0",
    "tsconfig-paths": "^3.9.0"
  },
  "devDependencies": {
    "@barusu/eslint-config": "^0.0.31",
    "@barusu/rollup-config": "^0.0.31",
    "rollup": "2.16.1"
  }
}
