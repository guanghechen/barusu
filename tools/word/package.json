{
  "name": "@barusu/tool-word",
  "version": "0.1.1",
  "description": "Count the frequency of characters in the file(s)",
  "author": {
    "name": "guanghechen",
    "url": "https://github.com/guanghechen/"
  },
  "repository": {
    "type": "git",
    "url": "https://github.com/guanghechen/barusu.git",
    "directory": "tools/word"
  },
  "homepage": "https://github.com/guanghechen/barusu/tree/master/tools/word#readme",
  "main": "lib/cjs/index.js",
  "types": "lib/types/index.d.ts",
  "source": "src/index.ts",
  "license": "MIT",
  "engines": {
    "node": ">=14.15.0"
  },
  "files": [
    "lib/",
    "!lib/**/*.js.map",
    "!lib/**/*.d.ts.map",
    "package.json",
    "CHANGELOG.md",
    "LICENSE",
    "README.md"
  ],
  "bin": {
    "barusu-word": "./lib/cjs/cli.js"
  },
  "scripts": {
    "build": "cross-env NODE_ENV=production rollup -c ../../rollup.config.cli.js",
    "prebuild": "rimraf lib/",
    "prepublishOnly": "cross-env ROLLUP_SHOULD_SOURCEMAP=false yarn build",
    "test": "cross-env TS_NODE_FILES=true jest --config ../../jest.config.js --rootDir ."
  },
  "dependencies": {
    "@barusu/chalk-logger": "^0.1.1",
    "@barusu/event-bus": "^0.1.1",
    "@barusu/util-cli": "^0.1.1",
    "@guanghechen/option-helper": "^1.2.1",
    "@types/fs-extra": "^9.0.11",
    "commander": "^7.2.0",
    "fs-extra": "^10.0.0",
    "globby": "^11.0.3"
  }
}
