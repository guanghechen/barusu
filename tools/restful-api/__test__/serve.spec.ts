import fs from 'fs-extra'
import path from 'path'
import rimraf from 'rimraf'
import supertest from 'supertest'
import { name } from '@barusu/tool-restful-api/package.json'
import { absoluteOfWorkspace } from '@barusu/util-cli'
import Router from '@koa/router'
import {
  COMMAND_NAME,
  RestfulApiServeContext,
  RestfulApiServeProcessor,
  SubCommandServeOptions,
  createProgram,
  createRestfulApiServeContext,
  createSubCommandServe,
} from '../src'


describe('serve', function () {
  const caseRootDirectory = path.resolve(__dirname, 'cases', 'generate')
  const kases = fs.readdirSync(caseRootDirectory)

  for (const kase of kases) {
    const title = kase
    const projectDir = absoluteOfWorkspace(caseRootDirectory, kase)

    // clean
    const logRootDir = '__tmp__/logs/serve'
    const absoluteLogRootDir = absoluteOfWorkspace(projectDir, logRootDir)
    rimraf.sync(absoluteLogRootDir)

    test(title, async function () {
      const program = createProgram()
      const promise = new Promise<RestfulApiServeContext>(resolve => {
        program.addCommand(createSubCommandServe(
          name,
          async (options: SubCommandServeOptions): Promise<void> => {
            const context: RestfulApiServeContext = await createRestfulApiServeContext({
              cwd: options.cwd,
              workspace: options.workspace,
              tsconfigPath: options.tsconfigPath,
              schemaRootPath: options.schemaRootPath,
              apiConfigPath: options.apiConfigPath,
              encoding: options.encoding,
              host: options.host,
              port: options.port,
              prefixUrl: options.prefixUrl,
              mockRequiredOnly: options.mockRequiredOnly,
              mockOptionalsAlways: options.mockOptionalsAlways,
              mockOptionalsProbability: options.mockOptionalsProbability,
              mockDataPrefixUrl: options.mockDataPrefixUrl,
              mockDataRootDir: options.mockDataRootDir,
              mockResourcePrefixUrl: options.mockResourcePrefixUrl,
              mockResourceRootDir: options.mockResourceRootDir,
            })
            resolve(context)
          }
        ))
      })

      const args = [
        '',
        COMMAND_NAME,
        'serve',
        projectDir,
        '--log-level=debug',
        '--config-path=app.yml',
        '--api-config-path=api.yml',
        '--schema-root-path=schemas/answer',
      ]

      program.parse(args)

      const context = await promise

      const router = new Router()
      router.get(context.prefixUrl + '/uu/vv', ctx => {
        console.log('okay')
        // eslint-disable-next-line no-param-reassign
        ctx.body = {
          code: 200,
          message: 'Got it!',
        }
      })

      const server = new RestfulApiServeProcessor(context)
      server.registerRouter(router)
      await server.start()


      // waiting the program finished
      const request = supertest(server.app!.callback())

      const requestItem = [
        { url: context.prefixUrl + '/uu/vv', method: 'get' },
      ]
      for (const item of requestItem) {
        const response = await request[item.method](item.url)
        expect({
          status: response.status,
          text: response.text,
        }).toMatchSnapshot(item.url + '__' + item.method)
      }

      // close server
      await server.stop()
    })
  }
})
