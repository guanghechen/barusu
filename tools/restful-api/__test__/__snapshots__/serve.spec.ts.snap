// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`serve simple: /mock/uu/vv__get 1`] = `
Object {
  "status": 200,
  "text": "{\\"code\\":200,\\"message\\":\\"Got it!\\"}",
}
`;
